source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/imu/imu_cfg.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+from __future__ import annotations
+
+from omni.isaac.lab.markers import VisualizationMarkersCfg
+from omni.isaac.lab.markers.config import RED_ARROW_X_MARKER_CFG
+from omni.isaac.lab.utils import configclass
+
+from ..sensor_base_cfg import SensorBaseCfg
+from .imu import Imu
+
+
+@configclass
+class ImuCfg(SensorBaseCfg):
+    """Configuration for an inertial measurement unit (Imu) sensor."""
```

Comment:
NIT: all docstrings could use "IMU" instead of "imu" or "Imu"

```suggestion
    """Configuration for an Inertial Measurement Unit (IMU) sensor."""
```